Automattic/jetpack
# Stats: Reduce CTAs

Per pbNhbs-1lh-p2 , let's remove low-performing CTAs. 

#### Changes proposed in this Pull Request:

* Remove scan and akismet nudges.

#### Jetpack product discussion


pbNhbs-1lh-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to Jetpack->Site Stats on a connected page. 
* See the CTAs. You shouldn't see anti-spam and scan.
